### ISSUE TITLE
enqueue conflict with preempt

### ISSUE THREAD

**Issue by hzxuzhonghu** [volcano-sh/volcano#991]
**Is this a BUG REPORT or FEATURE REQUEST?**:

 /kind bug

**What happened**:

Create a low priority job to occupy all the cluster resource, then create a high priority job, we can find that the high priority job can never become `inqueue` status.

This is actually a regression since enqueue feature introduced. 

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Volcano Version:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:

**Comment by hzxuzhonghu** [comment]
cc @william-wang

**Comment by hzxuzhonghu** [comment]
I donot think it is worth coupling enqueue with preempt/reclaim. Currently, i want to declare it in the release note.  If users need preempt, enqueue should be disabled.

**Comment by Thor-wl** [comment]
This bug will be fix in v1.1.1 in late Dec
